src/content-steering-controller.js

```diff
@@ -139,18 +136,18 @@ export default class ContentSteeringController extends videojs.EventTarget {
    *    This scenario is specific to DASH when the queryBeforeStart parameter is true.
    *    This scenario should only happen once on initalization.
    */
-  requestSteeringManifest(initialUri) {
+  requestSteeringManifest() {
```

Comment:
I think my reasoning for including this `initialUri` logic is due to the spec saying that in the cases of `queryBeforeStart = true`, we are supposed to omit the `_DASH_pathway` and `_DASH_throughput` values on the initial request to the steering server. Is that still handled?

If we do want to remove this, I think we want to make sure we remove that param from the documentation as well.

Reply:
I was thinking we just send them if we have them, but I looked at the DASH spec again and you're correct. My mistake, fix incoming!